includes/class-sensei-course-enrolment-provider-results.php

```diff
@@ -105,12 +105,6 @@ public function is_enrolment_provided() {
 			return true;
 		}
 
-		// @todo Remove this once we have a core provider, such as manual enrolment.
-		// If there are no providers, return `null` and let Sensei handle it.
-		if ( empty( $provider_results ) ) {
```

Comment:
Just so I understand why we don't need this:

- We were only using it to detect whether we needed to use the legacy system.
- If, for example, someone filtered the providers such that no provider handles this enrolment, making `$provider_results` empty, this method would return `false` instead of `null`. And we're ok with that, right?

Reply:
Yes, this makes it so if they filter out all providers, this would return `false` always. If no one is providing enrolment, enrolment should not be given. The manual enrolment provider is now should be the only "legacy aware" piece of this puzzle.